cmd/contour/serve.go

```diff
@@ -1124,24 +1126,27 @@ func (s *Server) getDAGBuilder(dbc dagBuilderConfig) *dag.Builder {
 			ConnectTimeout:    dbc.connectTimeout,
 		},
 		&dag.HTTPProxyProcessor{
-			EnableExternalNameService:   dbc.enableExternalNameService,
-			DisablePermitInsecure:       dbc.disablePermitInsecure,
-			FallbackCertificate:         dbc.fallbackCert,
-			DNSLookupFamily:             dbc.dnsLookupFamily,
-			ClientCertificate:           dbc.clientCert,
-			RequestHeadersPolicy:        &requestHeadersPolicy,
-			ResponseHeadersPolicy:       &responseHeadersPolicy,
-			ConnectTimeout:              dbc.connectTimeout,
-			GlobalExternalAuthorization: dbc.globalExternalAuthorizationService,
-			MaxRequestsPerConnection:    dbc.maxRequestsPerConnection,
+			EnableExternalNameService:     dbc.enableExternalNameService,
+			DisablePermitInsecure:         dbc.disablePermitInsecure,
+			FallbackCertificate:           dbc.fallbackCert,
+			DNSLookupFamily:               dbc.dnsLookupFamily,
+			ClientCertificate:             dbc.clientCert,
+			RequestHeadersPolicy:          &requestHeadersPolicy,
+			ResponseHeadersPolicy:         &responseHeadersPolicy,
+			ConnectTimeout:                dbc.connectTimeout,
+			GlobalExternalAuthorization:   dbc.globalExternalAuthorizationService,
+			MaxRequestsPerConnection:      dbc.maxRequestsPerConnection,
+			PerConnectionBufferLimitBytes: dbc.perConnectionBufferLimitBytes,
```

Comment:
Need to set these on the `IngressProcessor` above as well

Reply:
Fixed